src/components/ButtonDropdown.tsx

```diff
@@ -120,7 +119,7 @@ export interface IProps {
 
 const ButtonDropdown: React.FC<IProps> = ({ list, className }) => {
   const [isOpen, setIsOpen] = useState<boolean>(false)
-  const intl = useIntl()
+  const { t } = useTranslation()
```

Comment:
This isnt used anywhere now, I think we can just remove this from the file altogether